libs/blog-bff/shared/util-wp/src/lib/wp-subscribers-middleware.ts

```diff
@@ -0,0 +1,26 @@
+import { createMiddleware } from 'hono/factory';
+
+import { WPRestClient } from './wp-client';
+
+export const wpClientSubscribersMw = createMiddleware<{
+  Bindings: {
+    WP_REST_API_BASE_URL: string;
+    CLIENT_KEY: string;
+    CLIENT_SECRET: string;
+  };
+  Variables: { wpClientSubscriber: WPRestClient };
+}>(async (c, next) => {
+  const client_config = btoa(`${c.env.CLIENT_KEY + ':' + c.env.CLIENT_SECRET}`);
+  const token = `Basic ${client_config}`;
+  if (!token) throw 'Token is not defined!';
```

Comment:
it will never be thrown.
```ts
  const token = `Basic ${client_config}`;
```
^it will always have a value even if client_config is undefined. but also:
```ts
btoa(`${undefined + ':' + undefined}`);
``` 
returns a string value